OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -0,0 +1,308 @@
+
+#### A. Data on a volume for Elasticsearch Container
+We need to mount data volume to store crawled data outside the Elasticsearch container.
+
+To mount a host directory `/scratch/wcpdocker/volumes/es` ($ES_DATA_VOLUME) as a data volume, execute the below command.
+
+```
+$ sudo mkdir -p /scratch/wcpdocker/volumes/es
+$ sudo chown 1000:1000 /scratch/wcpdocker/volumes/es
+```
+
+#### B. Creating and Running Elasticsearch Container
+
+```
+$ docker run -i -t --name ESContainer --network=WCPortalNET -p 9200:9200 --volumes-from WCPortalContainer -v $ES_DATA_VOLUME:/u01/esHome/esNode/data --env-file <directory>/webcenter.env.list $WCPortalImageName /bin/bash -c "/u01/oracle/container-scripts/configureOrStartElasticsearch.sh; /bin/bash"
+```
+
+#### C. Stopping Elasticsearch Container
+```
+$ docker container stop ESContainer
+```
+
+#### D. Starting Elasticsearch Container
+```
+$ docker container start -i ESContainer
+```
+
+#  FAQs
+
+
+##### 1. How do I reuse existing domain?
+To reuse a existing domain we need to comply with existing domain configuration given in `webcenter.env.list` and container names.
+
+Follow below instruction to recreate a container in such a way that they continue using existing domain on data volume.
+
+*Delete existing container (ESContainer,WCPortalContainer and WCPAdminContainer)*
+ 
+```
+$ docker container stop ESContainer
+$ docker container stop WCPortalContainer
+$ docker container stop WCPAdminContainer
+
+$ docker container rm ESContainer
+$ docker container rm WCPortalContainer
+$ docker container rm WCPAdminContainer
+
+```
+
+*Reuse the WebCenter Portal database schema*
+
+Update the following parameter inside `webcenter.env.list` :
+
+```
+DB_DROP_AND_CREATE=false
+
+```
+
+*Create container pointing to existing domain*
+*Run the command to create Container in given Sequence (run as non root user)*
+```
+# create Admin Container
+$ docker run -i -t --name WCPAdminContainer --network=WCPortalNET -p 7001:7001 -v /scratch/wcpdocker/volumes/wcpportal:/u01/oracle/user_projects --env-file /scratch/<userid>/docker/webcenter.env.list oracle/wcportal:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/configureOrStartAdminServer.sh; /bin/bash"
+ 
+# create Portal Container
+$ docker run -i -t --name WCPortalContainer --network=WCPortalNET -p 8888:8888 -v /scratch/wcpdocker/volumes/wcpportal:/u01/oracle/user_projects --env-file /scratch/<userid>/docker/webcenter.env.list oracle/wcportal:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/configureOrStartWebCenterPortal.sh; /bin/bash"
+
+# create Elasticsearch Container
+$ docker run -i -t --name ESContainer --network=WCPortalNET -p 9200:9200 --volumes-from WCPortalContainer -v /scratch/wcpdocker/volumes/es:/u01/esHome/esNode/data --env-file /scratch/<userid>/docker/webcenter.env.list oracle/wcportal:12.2.1.4.0 /bin/bash -c "/u01/oracle/container-scripts/configureOrStartElasticsearch.sh; /bin/bash"
```

Comment:
Remove `/bin/bash` at least, but everything after the image name should really be removed.